frontend/feat/feature-flags.json

```diff
@@ -34,7 +34,7 @@
     "dark_mode_ui_toggle": {
       "status": {
         "staging": "switchable",
-        "production": "disabled"
+        "production": "switchable"
```

Comment:
This should be "enabled" now. Not switchable anymore.

Reply:
Great catch, thank you!